Dockerfile

```
@@ -1,19 +1,19 @@
 ############# builder
-FROM eu.gcr.io/gardener-project/3rd/golang:1.16.2 AS builder
+FROM eu.gcr.io/gardener-project/3rd/golang:1.16.6 AS builder
```

Since some months the Concourse is using kaniko as image building and now the request to docker hub are authenticated, i.e. the issue with rate limits is fixed. So you can use the upstream image
```suggestion
FROM golang:1.16.6 AS builder
```
```

---

Thanks for the hint, adapted it.